app/assets/javascripts/qz/ticket-popup.js

```diff
@@ -0,0 +1,85 @@
+var privateKey = "-----BEGIN RSA PRIVATE KEY-----\n" +
```

Comment:
The other issue (that @RohanM pointed out to me) is that we are assuming that "print(ing) things via qz" is actually the worst thing that an attacker can do with our private key. I have no knowledge of the capabilities of the Qz client, and so I have no way of knowing whether this is a good assumption or not, especially seeing as we are breaking their "security model"...